gtk/text_child_anchor.go

```diff
@@ -17,3 +22,29 @@ type TextChildAnchor C.GtkTextChildAnchor
 func (v *TextChildAnchor) native() *C.GtkTextChildAnchor {
 	return (*C.GtkTextChildAnchor)(v)
```

Comment:
It seems to me, according to [the documentation](https://developer.gnome.org/gtk3/stable/GtkTextView.html), that `TextChildAnchor` should be of type `struct` with a `glib.InitiallyUnowned` (similarly to a Widget), as it could contain references. `glib.Take` should also be used because of this.

Note that `gtk_text_buffer_insert_child_anchor`'s documentation suggests that the function itself takes a reference on its own, so we must unreference it once it's free, so `glib.Take` should be used.

Reply:
I'm ok,
I have not implemented the main structure, i just added methods for quick use.
Must be rewritten in correct way certainly.